### PR TITLE
fix: flight deck API update for RWY TAKE OFF switch pos

### PR DESCRIPTION
**Pull request by 2hwk**
### Location

Discord username (if different from GitHub):

### CODE REVIEW

File: docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Flight Deck:  [EXT LT Panel](../../pilots-corner/a32nx-briefing/flight-deck/ovhd
 |              |                       |          |            |                  |                                                                    |
 | RWY TURN OFF | CIRCUIT SWITCH ON:21  | 0&#124;1 | R/W        | MSFS VAR         | Left Rwy Turn Off Light + Switch                                   |
 |              | CIRCUIT SWITCH ON:22  | 0&#124;1 | R/W        | MSFS VAR         | Right Rwy Turn Off Light                                           |
-|              | LIGHTING_TAXI_2       | 0&#124;1 | R          | Custom LVAR      |                                                                    |
+|              | LIGHT TAXI:2          | 0&#124;1 | R/W        | SIMCONNECT VAR   | Rwy Turn Off Switch Position                                       |
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md]:
```suggestion
|              | LIGHT TAXI:2          | 0&#124;1 | R/W        | SIMCONNECT VAR   | Rwy Turn Off Light + Switch                                 |
```

Comment (frankkopp) [reply docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md]: Btw it is not only switch position - it does turn on/off lights (if ac is powered) - I usually have differentiated only if the switch is separated from the effect. So I'd say we the remark,

**Comment by Valastiri** [comment]
Safe to assume that one review edit and this is good to publish?

**Comment by frankkopp** [comment]
> Safe to assume that one review edit and this is good to publish?

Yea, even tested it. Works - but as SPAD does not yet support the Var as indexed we can/should leave the CIRCUITs as well.

**Comment by Valastiri** [comment]
I'm a bit confused -- do the lines in the PR need additional clarification as per slein's review or can this be merged?

**Comment by Sleinmaster** [comment]
Tbh I'm not sure about that either. In the lines before that the light is present and it's easier to read, but the label on the switch is just "rwy turn off", though on the ext lt panel.

Also the lines 105 and 106 are inconsistent and should be corrected in my opinion.

Me personally would add the light

**Comment by Valastiri** [comment]
> Tbh I'm not sure about that either. In the lines before that the light is present and it's easier to read, but the label on the switch is just "rwy turn off", though on the ext lt panel.
> 
> Also the lines 105 and 106 are inconsistent and should be corrected in my opinion.
> 
> Me personally would add the light

Could you update your review so and possibly commit it. I trust ya'll with this. I figured better to get this merged as more and more people reference our API docs so best to keep it current and such.

File: docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Flight Deck:  [EXT LT Panel](../../pilots-corner/a32nx-briefing/flight-deck/ovhd
 |              |                       |          |            |                  |                                                                    |
 | RWY TURN OFF | CIRCUIT SWITCH ON:21  | 0&#124;1 | R/W        | MSFS VAR         | Left Rwy Turn Off Light + Switch                                   |
 |              | CIRCUIT SWITCH ON:22  | 0&#124;1 | R/W        | MSFS VAR         | Right Rwy Turn Off Light                                           |
-|              | LIGHTING_TAXI_2       | 0&#124;1 | R          | Custom LVAR      |                                                                    |
+|              | LIGHT TAXI:2          | 0&#124;1 | R/W        | SIMCONNECT VAR   | Rwy Turn Off Light + Switch                                 |
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md]:
```suggestion
|              | LIGHT TAXI:2          | 0&#124;1 | R/W        | SIMCONNECT VAR   | Rwy Turn Off Light + Switch                                 
      |
```

File: docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md [hunk 3]
```diff
@@ -104,7 +104,8 @@ Flight Deck:  [EXT LT Panel](../../pilots-corner/a32nx-briefing/flight-deck/ovhd
 |              |                       |          |            |                  |                                                                    |
 | RWY TURN OFF | CIRCUIT SWITCH ON:21  | 0&#124;1 | R/W        | MSFS VAR         | Left Rwy Turn Off Light + Switch                                   |
 |              | CIRCUIT SWITCH ON:22  | 0&#124;1 | R/W        | MSFS VAR         | Right Rwy Turn Off Light                                           |
-|              | LIGHTING_TAXI_2       | 0&#124;1 | R          | Custom LVAR      |                                                                    |
+|              | LIGHT TAXI:2          | 0&#124;1 | R/W        | SIMCONNECT VAR   | Rwy Turn Off Light + Switch                                 
+      |
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/a32nx-api/a32nx-flightdeck-api.md]:
```suggestion
|              | LIGHT TAXI:2          | 0&#124;1 | R/W        | SIMCONNECT VAR   | Rwy Turn Off Light + Switch                                       |
```

**Pull request merged**